### DISCUSSION TITLE
夏日里的云朵 | /posts/see-cloud/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#621] [Announcements]
https://yuanfan.rbind.io/posts/see-cloud/

**Comment by hxglife** [comment]
看蓝天白云真有种心旷神怡的感觉，非常解压

**Reply by earfanfan** [reply]
其实我还拍到一些特别的，比如一千七百米高空居然还有飞鸟，分享与你，请你看哈。
![一千七百米高空翱翔的自由鸟](https://github.com/earfanfan/yuanfan/assets/29876335/2ae6bbe6-691c-4089-ba2e-1b1a8c0ff4a8)

**Reply by Liam-Zhong** [reply]
云无心以出岫，鸟倦飞而知还

**Comment by HushWay** [comment]
今天拍的云长这样，天上住着一位孩子神仙，他今天在练习吹泡泡！
![泡泡云](https://raw.githubusercontent.com/HushWay/Typora-img/main/img/泡泡云.jpg)
![泡泡云2](https://raw.githubusercontent.com/HushWay/Typora-img/main/img/泡泡云2.jpg)

**Reply by earfanfan** [reply]
哇，我从来没想到过这也可以是吹泡泡，看来云的魔法就是允许各人有各人的想象啊。我看到这两个照片后，第一反应的想象是酱紫的，天空上有另外一个世界，那个世界和我们这个世界拥有同一片天空，只是刚好相反，在另一个世界里正好有很多乌龟在爬，乌龟的壳透过两个世界共同的天空，就这样映照成了我们世界的白云。

**Reply by HushWay** [reply]
哈哈，你也很有想象力。如此说来，这个世界的白云是另一个世界的“甲骨文”，卜一卜今天是什么天气呢？

**Reply by earfanfan** [reply]
不卜不卜，哈，你没有明白，另一个世界并不只有乌龟，还有许多其他生物。但另一个世界是一个时间与空间均十分混乱的世界，并不像我们这个世界这样如此存在秩序和规律。在那个世界，有时候会有一大群乌龟爬过去，有时候又只是一些我们这个世界的人类脑子里想象过的物体漂浮着。所以，你该明白了，我们这个世界和另一个世界也存在某种连接，也许我们这个世界在三千年前曾有人想象过一种巨大的混沌生物，这一边的想象映照到了另一边，于是三千年后我们又再一次通过映照看到了三千年前的想象。啊，好绕，我也快绕晕了……

**Reply by HushWay** [reply]
哦哦，我看到的云原来是三千年的人想象出来的诡异世界的混沌生物在这个世界通过特殊连接建立的投影呀，仔细一想非（fei）！常（yi）！合（suo）！理（si）！呢！。
